ui/shared/Input.qml

```diff
@@ -90,6 +92,46 @@ Item {
             fillMode: Image.PreserveAspectFit
             source: inputBox.icon
         }
+
+        Loader {
+            active: inputBox.copyToClipboard
+            anchors.verticalCenter: parent.verticalCenter
+            anchors.right: parent.right
+            anchors.rightMargin: 8
+            sourceComponent: copyToClipboardComponent
+        }
+
+        Component {
+            id: copyToClipboardComponent
+
+            Item {
+                width: copyBtn.width
+                height: copyBtn.height
+
+                Timer {
+                    id: timer
+                }
+
+                StyledButton {
+                    id: copyBtn
+                    label: qsTr("Copy")
+                    height: 28
+                    textSize: 12
+                    btnBorderColor: Style.current.blue
+                    btnBorderWidth: 1
+                    onClicked: {
+                        chatsModel.copyToClipboard(inputValue.text)
+                        copyBtn.label = qsTr("Copied")
```

Comment:
That's cool for now. I think ideally we'll have something like a little tooltip on the copy button that indicates "copied" state, but we can do something like that later.

Reply:
Yeah you're right. That's what was the design, but I was lazy and we have no tooltip yet so I felt like it would just delay this PR a lot